src/useApi.js

```diff
@@ -0,0 +1,37 @@
+const useApi = () => {
+  const baseurl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+  const scoreurl = 'games/superawesomegame/scores/';
+  const outputdiv = document.querySelector('.scores');
+  const userinp = document.querySelector('.userinp');
+  const scoreinp = document.querySelector('.scoreinp');
+  const loaddata = () => {
+    axios.get(baseurl + scoreurl).then((response) => {// eslint-disable-line
+      outputdiv.innerHTML = '';
+      response.data.result.forEach((item) => {
+        outputdiv.innerHTML
+          += `<li>name:${item.user} score:${item.score}</li>`;
+      });
+    });
+  };
+
+  const adduser = () => {
+    console.log(userinp);
```

Comment:
- [ ] [OPTIONAL] Try to avoid committing `console.log` to your repo. it will make your code look ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.

Reply:
Thanks i missed that line, will remove it!

Reply:
Ja, ik zag dat je wat testen in het Nederlands had geschreven vandaar dat ik zag dat je Nederlander bent

Reply:
Eigenlijk uit Alkmaar, Ik zie dat jij uit Polen komt? woon je of heb je in Nederland gewoond? Voor de rest heb ik nog helemaal geen Nederlander gevonden in Microverse, jij wel? @mikethreels